kickstart/etc/nginx-posm.conf

```diff
@@ -24,18 +24,54 @@ server {
   }
 
   # proxied locations
+
+  location ~ ^(/osm)?/api/0.6/map$ {
+    include /etc/nginx/fastcgi_params;
+    fastcgi_pass 127.0.0.1:{{cgimap_port}};
+  }
+
+  location ~ ^(/osm)?/api/0.6/(nodes|ways|relations)$ {
+    include /etc/nginx/fastcgi_params;
+    fastcgi_pass 127.0.0.1:{{cgimap_port}};
+  }
+
+  location ~ ^(/osm)?/api/0.6/(way|relation)/([^/]+)/full$ {
+    include /etc/nginx/fastcgi_params;
+    fastcgi_pass 127.0.0.1:{{cgimap_port}};
+  }
+
+  location ~ ^(/osm)?(/api/0.6/(way|relation)/([^/]+))$ {
```

Comment:
Sorted in https://github.com/AmericanRedCross/posm-build/commit/ee3b8113330166e8b3d4a7e5fe8c5d7d228b9a10

`fastcgi_param` needed to be specified before `include /etc/nginx/fastcgi_params;`